plugins/generator-1.20.1/forge-1.20.1/block.definition.yaml

```diff
@@ -388,6 +388,20 @@ templates:
     writer: json
     name: "@MODASSETSROOT/models/block/@registryname_inventory.json"
 
+block_base_properties:
+  Stairs: [facing, half, shape, waterlogged]
+  Slab: [type, waterlogged]
+  Fence: [north, south, west, east, waterlogged]
+  Wall: [up, north, south, west, east, waterlogged]
+  Leaves: [distance, persistent, waterlogged]
+  TrapDoor: [facing, half, open, powered, waterlogged]
+  Pane: [north, south, west, east, waterlogged]
+  Door: [facing, half, hinge, open, powered]
+  FenceGate: [in_wall, open, powered]

```

Comment:
Still missing entries....

![image](https://github.com/user-attachments/assets/5bc0ca70-18a3-4933-a661-9d74cb151859)

Please go through the list again and make sure to not miss any this time, please..

Reply:
Seems there are no more properties.